### PR TITLE
Update Project Lombok version to 1.16.20

### PR DESCRIPTION
**Pull request by deathbeam**
Update Project Lombok version to 1.16.20 (from 1.16.18) and specify this
version in runelite-parent instead of having it specified all around the
maven projects.

Signed-off-by: Tomas Slusny <slusnucky@gmail.com>

### CODE REVIEW

**Comment by deathbeam** [comment]
Confirmed to work with Java 9 by vvboy.

**Comment by vwboy** [comment]
works great with Java 9 no Issues whatsoever.

File: pom.xml [hunk 1]
```diff
@@ -129,6 +130,11 @@
 				<artifactId>guava</artifactId>
 				<version>23.2-jre</version>
 			</dependency>
+			<dependency>
+				<groupId>org.projectlombok</groupId>
+				<artifactId>lombok</artifactId>
+				<version>${lombok.version}</version>
```
Comment (Adam-) [inline pom.xml]: The managed version should be scope provided

Comment (deathbeam) [reply pom.xml]: Done

**Pull request merged**